### ISSUE TITLE
Use microphone for real claps and taps?

### ISSUE THREAD

**Issue by hallvors** [andyvn22/rhythm-game#1]
Hi @andyvn22, fun little thing you've made here :+1: 

Have you considered requesting access to the user's microphone to let the game support real claps and taps? 

(Naturally raises the question of how to distinguish "claps" and "taps" and background noise in code.. Should be a bit of a challenge but doable.)

**Comment by hallvors** [comment]
(I'm actually looking for something a group can play together - a single screen, all clapping. Although I find your space+enter keys solution clever and suprisingly engaging, I think a microphone mode would be fun too)

**Comment by andyvn22** [comment]
Yeah, this would be fantastic. If possible, the microphone should also be checking counts, which are currently ignored and just referenced repeatedly in the instructions. (I can say from experience that currently, _many_ students try to play without counting, and that many of them end up stuck as a result!)

**Comment by hallvors** [comment]
Great point, although complicated :) Even distinguishing "claps" from "taps" is going to be a bit complicated since you'd need to decide a volume threshold or something.. and it's definitely non-trivial to detect if the right number is spoken :)
